lib/cli/src/generators/baseGenerator.ts

```diff
@@ -51,6 +51,7 @@ const getFrameworkDetails = (
   builder?: string;
```

Comment:
@ndelangen — FYI, I ended up making a separate PR with this (slightly modified) change: https://github.com/storybookjs/storybook/pull/18968